backend/src/common/util/scheduled_workflow.go

```diff
@@ -126,3 +141,10 @@ func (s *ScheduledWorkflow) ToStringForStore() string {
 	}
 	return string(swf)
 }
+
+func (s *ScheduledWorkflow) IsV1() bool {
```

Comment:
Not a strong opinion, but I wonder if we should add a method IsV2() for v2 scheduledworkflows.

Reply:
Thank you. I changed this in response to Chen's suggestion above. You can call `GetVersion()` which covers v1, v2 and beyond.